src/components/discover-sheet/UniswapPoolsSection.js

```diff
@@ -55,32 +55,32 @@ const ErrorMessage = ({ colors, children }) => (
 const PoolEmoji = styled(Text).attrs({
   size: 'large',
   weight: 'heavy',
-})`
-  margin-top: ${android ? 0 : -4};
-`;
+})({
+  marginTop: android ? 0 : -4,
+});
 
 const PoolListButton = styled(ButtonPressAnimation).attrs({
   scaleTo: 0.96,
-})`
-  margin-right: 16px;
-  ${({ selected, theme: { colors }, titleColor }) =>
-    selected
-      ? `
-        background-color: ${colors.alpha(titleColor, 0.06)};
-        border-radius: 12px;
-        height: 30px;
-        padding-horizontal: 8px;
-        padding-top: ${ios ? 6.5 : 4.5}px;
-      `
-      : `
-        padding-top: ${ios ? 6.5 : 4.5}px;
-      `}
-`;
+})(({ selected, theme: { colors }, titleColor }) =>
+  selected
+    ? {
+        backgroundColor: colors.alpha(titleColor, 0.06),
+        borderRadius: 12,
+        height: 30,
+        marginRight: 16,
+        paddingHorizontal: 8,
+        paddingTop: ios ? 6.5 : 4.5,
+      }
+    : {
+        marginRight: 16,
```

Comment:
This is duplicated across both states, so once it's been hoisted, this can be deleted.